src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -5,58 +5,55 @@
       "Size": 3032
     },
     "assemblies/Java.Interop.dll": {
-      "Size": 54996
+      "Size": 55166
     },
     "assemblies/Mono.Android.dll": {
-      "Size": 88096
+      "Size": 88649
     },
     "assemblies/rc.bin": {
       "Size": 1083
     },
     "assemblies/System.Linq.dll": {
-      "Size": 10113
+      "Size": 10115
     },
     "assemblies/System.Private.CoreLib.dll": {
-      "Size": 519420
-    },
-    "assemblies/System.Runtime.CompilerServices.Unsafe.dll": {
-      "Size": 1163
+      "Size": 538104
     },
     "assemblies/System.Runtime.dll": {
-      "Size": 2367
+      "Size": 2417
     },
     "assemblies/UnnamedProject.dll": {
-      "Size": 3552
+      "Size": 3549
     },
     "classes.dex": {
       "Size": 345328
     },
     "lib/arm64-v8a/libmonodroid.so": {
-      "Size": 382200
+      "Size": 382960
     },
     "lib/arm64-v8a/libmonosgen-2.0.so": {
-      "Size": 3171952
+      "Size": 4699792
```

Comment:
@akoeplinger this was the size difference I noticed. `libmonosgen-2.0.so` does seem to be ~1.5MB larger?

Here is the .apk: [com.xamarin.buildreleasearm64-Signed.zip](https://github.com/xamarin/xamarin-android/files/8381162/com.xamarin.buildreleasearm64-Signed.zip)